sound/soc/sof/intel/hda-dsp.c

```diff
@@ -446,6 +464,9 @@ static int hda_dsp_set_D0_state(struct snd_sof_dev *sdev,
 		    !hda_enable_trace_D0I3_S0 ||
 		    sdev->system_suspend_target != SOF_SUSPEND_NONE)
 			flags = HDA_PM_NO_DMA_TRACE;
+
+		if (hda_dsp_d0i3_playback_active(sdev))
+			flags |= HDA_PM_PG_STREAMING;
```

Comment:
This patch could be the first one in the series or even a separate one... ?

Reply:
Thanks! I will put it into first one since it is related to d0i3 setting.